<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrokeStyleWidget</class>
 <widget class="QWidget" name="StrokeStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>282</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Style</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="spin_stroke_width">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Stroke Width</string>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cap</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="layout_cap">
         <item>
          <widget class="QToolButton" name="button_cap_butt">
           <property name="toolTip">
            <string>Butt</string>
           </property>
           <property name="text">
            <string>Butt</string>
           </property>
           <property name="icon">
            <iconset theme="stroke-cap-butt"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="button_cap_round">
           <property name="toolTip">
            <string>Round</string>
           </property>
           <property name="text">
            <string>Round</string>
           </property>
           <property name="icon">
            <iconset theme="stroke-cap-round"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="button_cap_square">
           <property name="toolTip">
            <string>Square</string>
           </property>
           <property name="text">
            <string>Square</string>
           </property>
           <property name="icon">
            <iconset theme="stroke-cap-square"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Join</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="layout_join">
         <item>
          <widget class="QToolButton" name="button_join_bevel">
           <property name="toolTip">
            <string>Bevel</string>
           </property>
           <property name="text">
            <string>Bevel</string>
           </property>
           <property name="icon">
            <iconset theme="stroke-join-bevel"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="button_join_round">
           <property name="toolTip">
            <string>Round</string>
           </property>
           <property name="text">
            <string>Round</string>
           </property>
           <property name="icon">
            <iconset theme="stroke-join-round"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="button_join_miter">
           <property name="toolTip">
            <string>Miter</string>
           </property>
           <property name="text">
            <string>Miter</string>
           </property>
           <property name="icon">
            <iconset theme="stroke-join-miter"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spin_miter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Miter Limit</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Dashes</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>128</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_join_miter</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_miter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cap_butt</sender>
   <signal>clicked()</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>check_cap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>-47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cap_round</sender>
   <signal>clicked()</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>check_cap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>-65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cap_square</sender>
   <signal>clicked()</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>check_cap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_join_miter</sender>
   <signal>clicked()</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>check_join()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_join_bevel</sender>
   <signal>clicked()</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>check_join()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>-23</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_join_round</sender>
   <signal>clicked()</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>check_join()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_stroke_width</sender>
   <signal>valueChanged(double)</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_miter</sender>
   <signal>valueChanged(double)</signal>
   <receiver>StrokeStyleWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check_join()</slot>
  <slot>check_cap()</slot>
 </slots>
</ui>
